packages/data-context/src/sources/FileDataSource.ts

```diff
@@ -9,6 +9,10 @@ import { toPosix } from '../util/file'
 
 const debug = Debug('cypress:data-context:sources:FileDataSource')
 
+export const matchGlobs = async (globs: string | string[], globbyOptions?: GlobbyOptions) => {
```

Comment:
An async function with `return await` is equivalent to a non-async function that just returns the promise.